main.go

```diff
@@ -209,6 +209,9 @@ Required unless running from provider root and set in upgrade-config.yml.`)
 	cmd.PersistentFlags().StringVar(&context.PrReviewers, "pr-reviewers", "",
 		`A comma separated list of reviewers to assign the upgrade PR to.`)
 
+	cmd.PersistentFlags().StringVar(&context.PrAssign, "pr-assign", "@me",
+		`A user to assign the upgrade PR to (default: @me).`)
```

Comment:
The default is already included by cobra, so this renders as 

```
--pr-assign string        A user to assign the upgrade PR to (default: @me). (default "@me")
```

```suggestion
	cmd.PersistentFlags().StringVar(&context.PrAssign, "pr-assign", "@me",
		`A user to assign the upgrade PR to.`)
```